server/lib/proxy.js

```diff
@@ -4,8 +4,8 @@
 
 /**
  * A proxy to forward requests to the auth server. Used for
- * older browsers like IE8 that do not have CORS enabled
- * XMLHttpRequest objects.
+ * older browsers that do not have CORS enabled XMLHttpRequest
```

Comment:
I believe the proxy files can be removed because the only reason they existed was IE8